### ISSUE TITLE
508-defect-2 [HEADINGS, COGNITION]: Headings must not look like regular body text

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#24162]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Details

The **When to expect a second dose** page has two headings "Moderna" and "Pfizer" that look like regular body text. This can create a situation where the section headers are missed when users skim the page. Screen shot attached below.

## Acceptance Criteria

- [ ] Moderna and Pfizer text remains H2's in markup, but styled as H3's (discussed this with Allison)

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/plan-second-dose

## WCAG or Vendor Guidance (optional)

* https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html
* https://www.tpgi.com/heading-off-confusion-when-do-headings-fail-wcag/

## Screenshots or Trace Logs

![Screen Shot 2021-05-04 at 8 36 00 AM](https://user-images.githubusercontent.com/934879/117013598-a14a6f00-acb5-11eb-87ac-8e4c9ce0eb37.png)

**Comment by outerpress** [comment]
Thanks @1Copenut! Just a couple of questions/clarifications:

>Moderna and Pfizer text remains H2's in markup, but styled as H3's (discussed this with Allison)
We'll need to look at a slightly different solution. At this stage, the user doesn't know which vaccine they will be getting, so highlighting the brand distracts from the important info. I'll put something together.

Also, is the posted screenshot meant for another ticket, or does this one cover the alert heading change too?

**Comment by 1Copenut** [comment]
@outerpress I realize I uploaded the wrong image. I removed the old one and added the right one for this ticket. My apologies!

**Comment by outerpress** [comment]
@1Copenut Thanks! I made a few other tweaks to reduce the noise on the page and focus attention on the important info. I still think this overemphasizes the brand a bit as they won't know which one they're getting at this stage, but it's not too bad. Let me know what you think.

![HP-2doses.png](https://images.zenhubusercontent.com/5f7ce3163782a585b910eaf3/54829e9f-fc8c-4a4e-9d73-938590753f60)

Changes
* Header styling to H3 as recommended
* Removed gray dividing lines
* Bolded only "after {date}"

**Comment by 1Copenut** [comment]
@outerpress I like this a lot. I'm able to scan the headings quickly and get down to vaccine types and dates.

**Comment by outerpress** [comment]
Great! Thanks @1Copenut 

@laurenernest this is GTG from my end

**Comment by jenniemc** [comment]
@outerpress, besides the styling change, is there also a change in the h1 copy and text copy? Did you want to lift the alert message language from the Choose Location page to the Second Dose page?

**Update h1 heading from** "When to expect a second dose" 
**To** "Some COVID-19 vaccine require 2 doses"

**Update text copy from** "You’ll need to return to the {facility.name} after the dates below, depending on which vaccine you receive:" 
**To** "If you get a vaccine that requires 2 doses, you'll need to return to the same facility for your second dose."

**Comment by outerpress** [comment]
@jenniemc nope, just the styling changes

**Comment by laurenernest** [comment]
@1Copenut ready for your review

**Comment by 1Copenut** [comment]
@laurenernest LGTM!
